backend/src/main/java/ch/zhaw/pm4/compass/backend/exception/DayNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package ch.zhaw.pm4.compass.backend.exception;
+
+import java.util.Date;
+
+public class DayNotFoundException extends RuntimeException {
```

Comment:
```suggestion
public class DaySheetNotFoundException extends RuntimeException {
```

Bitte auch anderswo korrigieren.